app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardPresenter.kt

```diff
@@ -19,20 +19,24 @@ import javax.inject.Inject
 class ConceptCardPresenter @Inject constructor(
   private val fragment: Fragment,
   private val viewModelProvider: ViewModelProvider<ConceptCardViewModel>
-){
+) {
 
   /** Sets up data binding and adapter for RecyclerView */
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?, skillId: String): View? {
-    val viewModel = getConceptCardViewModel()
-    viewModel.setSkillId(skillId)
     val binding = ConceptCardFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+
+    val viewModel = getConceptCardViewModel()
+    viewModel.setSkillIdAndBinding(skillId, binding)
+
     binding.conceptCardToolbar.setNavigationIcon(R.drawable.ic_close_white_24dp)
     binding.conceptCardToolbar.setNavigationOnClickListener {
       (fragment as? DialogFragment)?.dismiss()
     }
-    binding.workedExamples.apply {
-      adapter = createRecyclerViewAdapter()
-    }
+
+    // TODO(#352): Show worked examples in Concept Card.
+    // binding.workedExamples.apply {
```

Comment:
Similar to later, please remove commented out code.

Reply:
Ditto as earlier.